package-lock.json

```diff
@@ -5570,14 +6405,12 @@
 						"balanced-match": {
 							"version": "1.0.0",
 							"bundled": true,
-							"dev": true,
-							"optional": true
+							"dev": true
 						},
 						"brace-expansion": {
 							"version": "1.1.11",
 							"bundled": true,
 							"dev": true,
-							"optional": true,
```

Comment:
I'm guessing you installed these new dependencies using an older version of NPM. These changes shouldn't be here, and are causing local changes when running `npm install` from a fresh clone. Make sure to run the latest version of NPM when developing in Gutenberg ([reference](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/getting-started.md)).

Reply:
>and are causing local changes when running npm install from a fresh clone

This is addressed in #16567

Reply:
Thanks for the heads-up!
Updated my local npm version to avoid this in the future.

Sorry for the inconvenience 🙏